pkgs/top-level/all-packages.nix

```diff
@@ -42346,4 +42346,6 @@ with pkgs;
   ssl-proxy = callPackage ../tools/networking/ssl-proxy { };
 
   code-maat = callPackage ../development/tools/code-maat {};
+
+  gnoll = callPackage ../games/gnoll {};
```

Comment:
There is a separate category for games, starting on line 37511, where according to the top-of-file comment the new entry is supported to go in sorted. (But it's not consistent, so it's easy to get the impression that it's just unsorted.)